src/lib/eas/fetchAttestation.tsx

```diff
@@ -0,0 +1,31 @@
+import getDecodedAttestationData from "@/lib/eas/getDecodedAttestationData";
+import MockData from "@/components/Project/project.json";
+import { getAddress } from "@/lib/eas/getAddress";
+
+export const fetchAttestation = async (
+  projects: any,
```

Comment:
why are you passing the full list of projects into a function called `getAttestation`?
- actual: feels like `random` and `unrelated` data is being passed into this function
- expected: functions are `as small as possible` and only require the `minimum parameters` needed to accomplish the function objective